cmd/auth-handler.go

```diff
@@ -243,15 +243,6 @@ func getClaimsFromToken(token string) (map[string]interface{}, error) {
 		return claims.Map(), nil
 	}
 
-	// Session token must have a policy, reject requests without policy
-	// claim.
-	_, pokOpenIDClaimName := claims.MapClaims[iamPolicyClaimNameOpenID()]
-	_, pokOpenIDRoleArn := claims.MapClaims[roleArnClaim]
-	_, pokSA := claims.MapClaims[iamPolicyClaimNameSA()]
-	if !pokOpenIDClaimName && !pokOpenIDRoleArn && !pokSA {
-		return nil, errAuthentication
-	}
-
```

Comment:
are we okay to remove these? @donatello

Reply:
Yes, we don't need this - the main claim we need to check from the session token is if it is not already expired - this we already do. The policy for an STS credential is either from ParentUser or from RoleARN - we look for both in the IsAllowedSTS function.